src/components/dashboard/Dashboard.js

```diff
@@ -317,41 +313,6 @@ const Dashboard = props => {
     ],
   }).current
 
-  useEffect(() => {
```

Comment:
why all this been removed ?
actually you had to remove `animateMarket` state var only (and all references to it)

Reply:
e.g. remove L154, remove L352 and remove setAnimateMarket from the reps at L353

Reply:
Ye this should not have been removed, put back now